### DISCUSSION TITLE
How to Disable Price Cap

### DISCUSSION THREAD

**Discussion by comm** [BottlecapDave/HomeAssistant-OctopusEnergy#568] [Q&A] [answered]
Can anyone advise how I disable the Price Cap please? When I try to clear it through the Configure Dialogue, it says all updated, but opening the Configure dialogue shows it still set as previous value?

If I set it to a different figure, it changes and stays, but when clearing, it reverts back to what it was previously.

Thanks in advance!

**Comment by BottlecapDave** [comment, marked as answer]
There was a bug if you set a price cap when initially setting up the integration. This has now been resolved in v9.0.2

**Reply by comm** [reply]
Thank you!
